receiver/azureeventhubreceiver/azureresourcetraces_unmarshaler.go

```diff
@@ -0,0 +1,36 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package azureeventhubreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/azureeventhubreceiver"
+
+import (
+	eventhub "github.com/Azure/azure-event-hubs-go/v3"
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+	"go.uber.org/zap"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/azure"
+)
+
+type AzureTracesEventUnmarshaler struct {

```

Comment:
Try to not export this type

Reply:
Sorry I didn't get you , Can you please give me more context.

Reply:
```suggestion
type azureTracesEventUnmarshaler struct {

```

So it's not exposed as API.

Reply:
Hello @atoulme 
Changes were updated and rebased. Please approve the execute the workflow pipeline.